packages/sitecore-jss-rendering-host/src/renderingHostServer.ts

```diff
@@ -61,7 +61,7 @@ export function startRenderingHostServer({
       middlewares.compression && typeof middlewares.compression === 'function'
         ? middlewares.compression
         : compression();
-    server.use(compressionMiddleware);
+    server.use(compressionMiddleware as RequestHandler);
```

Comment:
I verified it locally and it works without type case, I guess nothing was changed, can you double check, please?

Reply:
Yes you're right. Maybe i reverted back a dependency upgrade and forgot to revert this change.